ts/parts-gantt/TreeGridAxis.ts

```diff
@@ -641,6 +641,15 @@ namespace TreeGridAxis {
                 }
             });
 
+            // If staticScale is not defined on the yAxis
+            // and chart height is set, set axis.isDirty
+            // to ensure collapsing works (#13601)
+            addEvent(axis, 'afterBreaks', function (): void {
+                if (axis.coll === 'yAxis' && !axis.staticScale && axis.chart.userOptions.chart?.height) {
```

Comment:
Use `axis.chart.options` instead of `userOptions`, to pick up settings from themes etc.